runtime/legion/region_tree.cc

```diff
@@ -7943,7 +7943,7 @@ namespace Legion {
       }
 #ifdef DEBUG_LEGION
       assert(layout != NULL);
-      assert(layout->constraints->specialized_constraint.is_file());
+   //   assert(layout->constraints->specialized_constraint.is_file());
```

Comment:
I don't know how to handle this assert properly. For external array, I just set the SpecializedConstraint(NORMAL_SPECIALIZE), so there is no way for me check it.